lib/node_modules/@stdlib/math/special/data/unary_function_database.json

```diff
@@ -0,0 +1,3386 @@
+{
+  "abs": {
+    "input_dtypes": "numeric_and_generic",
+    "output_dtypes": "real_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "int64",
+      "uint64",
+      "uint8c",
+      "complex32"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/labs",
+      "int16": "@stdlib/math/base/special/labs",
```

Comment:
In thinking about this, if we don't have dedicated kernels for particular dtypes (except for "generic"), we shouldn't include them in the list of scalar kernels.

In the generation script, I believe we try to find the nearest "compatible" scalar kernel based on type promotion, correct? In which case, for `int16` and `int8` here, we'd infer that `labs` should be used.